### PR TITLE
docs: add docs build kokoro config for django spanner

### LINKED ISSUE #643: Add documentation for django spanner.
Add documentation for django spanner.

### PR DESCRIPTION
**Pull request by vi3k6i5**
docs: added docs build kokoro config for django spanner.

fixes #643 

Docs will be published at link: https://googleapis.dev/python/django-google-spanner/latest/index.html

### CODE REVIEW

File: .repo-metadata.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "name": "django-google-spanner",
+  "name_pretty": "Cloud Spanner Django",
+  "product_documentation": "https://cloud.google.com/spanner/docs/",
+  "client_documentation": "https://googleapis.dev/django/spanner/latest",
+  "issue_tracker": "https://issuetracker.google.com/issues?q=componentid:190851%2B%20status:open",
+  "release_level": "ga",
+  "language": "python",
+  "library_type": "GAPIC_COMBO",
```
Comment (busunkim96) [inline .repo-metadata.json:9]: Per go/library-types:
```suggestion
  "library_type": "INTEGRATION",
```

File: .repo-metadata.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  "repo": "googleapis/python-spanner-django",
+  "distribution_name": "django-google-spanner",
+  "api_id": "django-google-spanner.googleapis.com",
```
Comment (busunkim96) [inline .repo-metadata.json:12]: This can be omitted since this library does not talk directly to a Google API endpoint

Comment (vi3k6i5) [reply .repo-metadata.json:12]: removed `"api_id": "django-google-spanner.googleapis.com"`, please let me know if any other key also needs to be removed example: `"requires_billing": true` ?

**Comment by vi3k6i5** [comment]
> Some small comments on `.repo-metadata.json`, otherwise LGTM. :)

@busunkim96 : Fixed the comments, Thanks for the help :)

**Comment by busunkim96** [comment]
I think you'll need to copy https://github.com/googleapis/python-spanner/blob/master/.trampolinerc into this repo as well. I triggered the docs job manually and it appears to be failing on a missing bucket name: [log](https://fusion.corp.google.com/projectanalysis/summary/KOKORO/prod:cloud-devrel%2Fclient-libraries%2Fpython%2Fgoogleapis%2Fpython-spanner-django%2Fdocs%2Fdocs)

**Comment by vi3k6i5** [comment]
> I think you'll need to copy https://github.com/googleapis/python-spanner/blob/master/.trampolinerc into this repo as well. I triggered the docs job manually and it appears to be failing on a missing bucket name: [log](https://fusion.corp.google.com/projectanalysis/summary/KOKORO/prod:cloud-devrel%2Fclient-libraries%2Fpython%2Fgoogleapis%2Fpython-spanner-django%2Fdocs%2Fdocs)

Added the same file, thanks

**Review by busunkim96 — APPROVED** [review]
Looks like the docs jobs is all good now! You should see them at googleapis.dev/python/django-spanner-google/latest in a bit. :) [docs job log](https://fusion2.corp.google.com/invocations/4c29346a-372c-4a9b-ae12-007670992c09/targets/cloud-devrel%2Fclient-libraries%2Fpython%2Fgoogleapis%2Fpython-spanner-django%2Fdocs%2Fdocs/log)

**Comment by busunkim96** [comment]
Docs are live! https://googleapis.dev/python/django-google-spanner/latest/index.html

FYI If you want to view the docs locally, you can run `nox -s docs` and open `docs/_build/html` in a browser.

**Review by larkee — APPROVED** [review]
I can see the docs at the given link so this LGTM :+1:

**Pull request merged**